data/soapvalidator/MailClient/GAL/ZCS-3390.xml

```diff
@@ -0,0 +1,221 @@
+
+        <t:test required="true">
+            <t:request>
+                <PingRequest xmlns="urn:zimbraAdmin" />
+            </t:request>
+            <t:response>
+                <t:select path="//admin:PingResponse" />
+            </t:response>
+        </t:test>
+
+    </t:test_case>
+
+
+    <t:test_case testcaseid="SyncGalRequest_AccountSetup_01" type="always">
+        <t:objective>Account Setup </t:objective>
+        <t:steps>
+            1. Login into admin.
+            2. Create test domains.
+            3. Create test galsync account.
+            4. Create accounts for the domain and sync it.
+        </t:steps>
+
+        <t:test id="adminlogin" required="true" depends="Ping">
+            <t:request>
+                <AuthRequest xmlns="urn:zimbraAdmin">
+                    <name>${admin.user}</name>
+                    <password>${admin.password}</password>
+                </AuthRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//admin:AuthResponse/admin:authToken" set="authToken" />
+            </t:response>
+        </t:test>
+
+        <t:test id="CreateDomain1a" required="true" depends="adminlogin">
+            <t:request>
+                <CreateDomainRequest xmlns="urn:zimbraAdmin">
+                    <name>${domain}</name>
+                    <a n="zimbraNotes">Domain for testing SyncGalRequest</a>
+                </CreateDomainRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//admin:CreateDomainResponse/admin:domain" attr="id" set="domain.id" />
+            </t:response>
+        </t:test>
+
+        <t:test>
+            <t:request>
+                <CreateGalSyncAccountRequest xmlns="urn:zimbraAdmin" name="${galdatasource}" type="zimbra"
+                    domain="${domain}" server="${zimbraServer.name}">
+                    <account by="name">${galaccount}@${domain}</account>
+                </CreateGalSyncAccountRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//admin:CreateGalSyncAccountResponse/admin:account" attr="id" set="galaccount.id" />
+            </t:response>
+        </t:test>
+
+        <t:test id="CreateAccount1a" required="true" depends="adminlogin">
+            <t:request>
+                <CreateAccountRequest xmlns="urn:zimbraAdmin">
+                    <name>${account1}@${domain}</name>
+                    <password>${defaultpassword.value}</password>
+                </CreateAccountRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//admin:CreateAccountResponse/admin:account" attr="id" set="account1.id" />
+                <t:select path='//admin:CreateAccountResponse/admin:account/admin:a[@n="zimbraMailHost"]' set="account1.server" />
+            </t:response>
+        </t:test>
+
+        <t:test id="CreateAccount1a" required="true" depends="adminlogin">
```

Comment:
Should the id be different to the one for the previous test?

Reply:
Every test is independent. Id does not matter. 
I will fix it.